content/en/docs/how-to/registry-authentication.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Authenticate with OCI-compliant registries"
+description: "How to use notation to authenticate with OCI-compliant registries"
+type: docs
+weight: 1
+---
+
+You can use `notation` to authenticate to an OCI-compliant registry. This is useful for pushing and pulling signed artifacts, as well as inspecting artifacts in the registry.
+
```

Comment:
Shall we add a prerequisite for this doc like the following sentence:

Currently, Notation relies on Docker Credential Helper to store the credentials in the local system key chain. You need to configure the [Docker Credential Helper](https://github.com/notaryproject/notation/blob/main/specs/registry-auth.md#credential-helper) before logging in to a registry. This dependency will be removed in the Notation v1.0.0-RC.4 release.

Reply:
@FeynmanZhou does this automatically work with docker desktop installed? or are there extra steps needed to install/configure?

I was able to test this flow using ACR without realizing Docker Credential Helper was involved.

Reply:
Yes, this automatically work with docker desktop installed. But without Docker Desktop, users will encounter this problem.

Reply:
fixed